release-team/shadows.md

```diff
@@ -0,0 +1,92 @@
+
+The idea is to make a lasting efficiency improvement, while leaving the Lead free to deal with current release issues.  Much of the existing Release Team infrastructure originated this way, and the ultimate aspiration of SIG-Release would be to automate away 80% of the work of the Release Team.
+
+Generally, if you're going to do an improvement project, you need to have a pretty good idea of what you want to do by the 2nd week of the release cycle.  You'll also need a knowledge of test-infra, docs/website, or some other relevant technical area where you're going to implement the improvement.  Finally, you'll want the agreement of your Role Lead and the Release Lead that the improvement you plan to implement (and how you want to implement it) is a good idea.
+
+## After Shadowing: What's Next?
+
+Shadowing is an apprenticeship, and as such is intended to be a stepping-stone to greater Kubernetes contributions in the future.  Towards the end of the release cycle, your Role Lead and the Emeritus Advisor (EA) will chat with you about what you want to do next.
+
+Here's the common paths for Shadows after the release cycle ends:
+
+**Shadowing Again**: if you didn't really get accustomed to the Role, possibly because personal issues interrupted Shadowing, you might want to Shadow the Role a second time.  Other Shadows discover that the first Role they tried isn't for them, and they want to shadow a different one so that they can try that one out.  In either case, you'll re-apply to Shadow when the next cycle opens, noting your experience in the application form.
+
+**Becoming Role Lead**: The main goal of the Shadow program is to train new Leads.  So, if you are ready to take on Leading the Role in the succeeding cycle, then let your Role Lead know.  They will be selecting a nominee or nominees to recommend to the incoming Release Lead, who has the final decision.  You can also skip a release cycle, and volunteer to become Role Lead in a later release cycle, in which case you should contact the incoming Release Lead as soon as they are announced.  There is usually an issue in the SIG-Release repository called "Assemble Kubernetes X.YY Release Team" where you can comment with your interest in a Role.
+
+**Another role in SIG-Release**: some Shadows go on to do something else within SIG-Release.  For example, they may join the Branch Managers as an RMA, or they may decide to take on a release-related project that's not directly tied to the release cycle.  If you are interested in the Branch Managers, wait for them to announce their next recruitment cycle, or ask in the SIG-Release Slack channel.
```

Comment:
what does RMA stand for?

Reply:
Release Management Associate.  Don't blame me, @justaugustus came up with the name.  ;-)

Reply:
I see - what are y'alls thoughts on spelling out the acronym here so new folks who aren't familiar with the term know what it means ?

Reply:
added to list of changes for next commit.